src/schema/schema_create.c

```diff
@@ -781,9 +781,14 @@ __schema_create(WT_SESSION_IMPL *session, const char *uri, const char *config)
     WT_CONFIG_ITEM cval;
     WT_DATA_SOURCE *dsrc;
     WT_DECL_RET;
-    bool exclusive;
+    bool exclusive, import;
 
     exclusive = __wt_config_getones(session, config, "exclusive", &cval) == 0 && cval.val != 0;
+    import = __wt_config_getones(session, config, "import.enabled", &cval) == 0 && cval.val != 0;
```

Comment:
Since I'm doing the check up here, I decided to pass `import` into the helpers instead of doing redundant configuration parsing later.

Reply:
I just saw this go by. This is a good thing. Configuration parsing is expensive so avoiding multiple calls to parse is better. Create is not a critical path but it does add up.